probe-rs/src/memory/adi_v5_memory_interface.rs

```diff
@@ -6,6 +6,8 @@ use crate::coresight::access_ports::{
 use crate::coresight::ap_access::APAccess;
 use scroll::Pread;
 
+use log::debug;
```

Comment:
can we just use log::debug!() everywhere instead of an use and then debug!()? For me this is way nicer. What do you folks think? It's not like log is long to write :)

Reply:
I think probe-rs uses imports way too infrequently in general. There's full paths everywhere.

Reply:
Well the paths sometimes help to make code more readable, because they give context.

Reply:
> I think probe-rs uses imports way too infrequently in general. There's full paths everywhere.

I agree to this fully. I still prefer that notation for logging. Anyways, I am not insisting on that, I just think we should streamline it and state my preference :)

Reply:
I prefer to have it not prefixed with a path because that makes it easier to switch do a different logging facade.